### PR TITLE
Initial Sample App

### PR DESCRIPTION
**Pull request by fjnoyp**
New PR for the Sample App. 

APIs and functionality updated.  Waiting on Storage API to be finalized. 

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by fjnoyp** [comment]
Outdate PR, switching to new PR here: https://github.com/aws-amplify/amplify-flutter/pull/51

File: README.md [hunk 1]
```diff
@@ -26,6 +26,9 @@ We are iterating and looking for feedback and collaboration, so please [**let us
  - Predictions
  - Datastore
  - Hub Events (Listening to the Amplify events)
+ - iOS Error Events in Amplify Analytics
+
+> Amplify for Flutter is in preview, which should be considered an alpha version and not recommended for production deployments. We are iterating and looking for feedback and collaboration, so please [**LET US KNOW YOUR FEEDBACK**](https://github.com/aws-amplify/amplify-flutter/issues/5) on our direction and roadmap.
```
[the remark this answers is not in the document]
Comment (fjnoyp) [reply README.md:31]: added

File: packages/examples/sample_app/lib/Views/ImageUploader.dart [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import 'package:flutter/material.dart';
+import 'package:file_picker/file_picker.dart';
+
+
+class ImageUploader extends StatelessWidget {
+  // how visually represent image to upload?
+
+  void _uploadImage(BuildContext context) async {
```
[the remark this answers is not in the document]
Comment (jamesonwilliams) [reply packages/examples/sample_app/lib/Views/ImageUploader.dart]: Sorry, this was a super noobie question, haha. Later, I read a basic Dart tutorial 😆  Thanks!

File: packages/examples/sample_app/android/app/src/main/kotlin/com/example/sample_app/MainActivity.kt [hunk 3]
```diff
@@ -0,0 +1,6 @@
+package com.example.sample_app
+
+import io.flutter.embedding.android.FlutterActivity
+
+class MainActivity: FlutterActivity() {
```
[the remark this answers is not in the document]
Comment (jamesonwilliams) [reply packages/examples/sample_app/android/app/src/main/kotlin/com/example/sample_app/MainActivity.kt:5]: Sounds good!

**Pull request closed without merging**